packages/contract/src/test-utils.ts

```diff
@@ -0,0 +1 @@
+export * from './test-utils/launch-test-node';
```

Comment:
1. We don't have a file naming convention and this file isn't put side-by-side with `launchNode.ts` so it doesn't poke out
2. Because it depends on classes like `ContractFactory` which are only available in `contract` and up.